mne_neuron/dipole.py

```diff
@@ -15,22 +17,44 @@ def _hammfilt(x, winsz):
     win /= sum(win)
     return convolve(x, win, 'same')
 
-
-def simulate_dipole(net):
+def simulate_dipole(fn):
     """Simulate a dipole given the experiment parameters.
 
     Parameters
     ----------
-    net : Network object
-        The Network object specifying how cells are
-        connected.
+    fn : string
+        The filename of the params file
 
     Returns
     -------
     dpl: instance of Dipole
         The dipole object
+    net : Network object
+        The Network object specifying how cells are
+        connected.
     """
-    pc = h.ParallelContext(1)
+    import mne_neuron
```

Comment:
This does not seem kosher to me. Typically you want to do relative imports within the package so you know where you are importing from. With an import like this, it could end up importing the wrong version of mne_neuron and it could lead to unexpected results.